internal/resource/nvml-device.go

```diff
@@ -99,3 +99,19 @@ func (d nvmlDevice) GetPCIClass() (uint32, error) {
 	}
 	return nvDevice.Class, nil
 }
+
+func (d nvmlDevice) GetFabricIds() (string, string, error) {
+	gfInfo, ret := d.GetGpuFabricInfo()
+	if ret != nvml.SUCCESS {
+		return "", "", ret
+	}
+
+	// ClusterUUID
+	byteSlice := gfInfo.ClusterUuid[:]
+	clusterUUID := string(byteSlice)
```

Comment:
Why not just:
```suggestion
	clusterUUID := string(gfInfo.ClusterUuid)
```

Reply:
The resulting label is `nvidia.com/gpu.clusteruuid:�Hg�/B��U���6~�`